lib/util.js

```diff
@@ -600,7 +600,7 @@ function formatNumber(fn, value) {
   // Format -0 as '-0'. A `value === -0` check won't distinguish 0 from -0.
   // Using a division check is currently faster than `Object.is(value, -0)`
   // as of V8 6.1.
-  if (1 / value === -Infinity)
+  if (value === 0 && 1 / value === -Infinity)
```

Comment:
`if (Object.is(value, -0))`

Reply:
The performance concern is real (3x slower) but does it really matter in this case? It feels like this be a micro-optimization too far.

Reply:
It got better recently but those changes may not be in 6.3 yet.  And yeah, I don't think `Object.is()` is going to be the expensive part here.